web-services/client/src/main/java/datawave/webservice/result/QueryWizardResultResponse.java

```diff
@@ -100,8 +101,8 @@ public String getMainContent() {
         builder.append("<div id=\"myTable_wrapper\" class=\"dataTables_wrapper no-footer\">\n");
         builder.append("<table id=\"myTable\" class=\"dataTable no-footer\" role=\"grid\" aria-describedby=\"myTable_info\">\n");
         
-        if (response instanceof DefaultEventQueryResponse) {
-            DefaultEventQueryResponse tempResponse = (DefaultEventQueryResponse) response;
+        if (response instanceof EdgeQueryResponseBase) {
```

Comment:
I suspect we want an `instanceof EventQueryResponseBase` here instead of an `instanceof EdgeQueryResponseBase`. Also check the end of this if statement and consider whether we want to throw an exception in the case where we don't know what to do with a specific event response type.

Reply:
doh... good catch